compliance_checker/base.py

```diff
@@ -32,30 +35,63 @@ def get_namespaces():
     ns["ows"] = n.get_namespace("ows110")
     return ns
 
+def csv_splitter(input_string):
+    """
+    csv_splitter(input_string)
+
+    Splits a string in CSV format and returns a flattened list
+
+    Parameters:
+    -----------
+    input_string: str
+        The string to be processed
+
+    Returns:
+    --------
+    list of str
+        A flattened list from the CSV processing contents
+    """
+    csv_contents = csv.reader(StringIO(input_string))
+    return list(itertools.chain.from_iterable(csv_contents))
+
 class ValidationObject(object):
```

Comment:
If this is intended to be an abstract base class, it might be beneficial to inherit from `abc.ABC ` or similar. Not critical.

Reply:
noted